client/analytics/report/customers/config.js

```diff
@@ -5,6 +5,7 @@
 import { __, _x } from '@wordpress/i18n';
 import { decodeEntities } from '@wordpress/html-entities';
 import { applyFilters } from '@wordpress/hooks';
+import { COUNTRIES as countries } from '@woocommerce/wc-admin-settings';
```

Comment:
Should we keep it in uppercase so it's clear that it's a constant when used in the code? That's what we are doing in WooCommerce Blocks.

Reply:
Ya, we _could_, I mostly was just looking to keep code churn to a minimum here.  Something for a future iteration?

Reply:
Sure thing! :+1: